consensus/src/experimental/tests/execution_phase_tests.rs

```diff
@@ -14,26 +17,32 @@ use futures::{
 };
 use std::sync::Arc;
 
-pub fn prepare_execution_phase() -> (
+pub fn prepare_execution_phase() -> (HashValue, ExecutionPhase) {
+    let execution_proxy = Arc::new(RandomComputeResultStateComputer::new());
+    let random_hash_value = execution_proxy.get_root_hash();
+    let execution_phase = ExecutionPhase::new(execution_proxy);
+    (random_hash_value, execution_phase)
+}
+
+pub fn prepare_execution_pipeline() -> (
     UnboundedSender<ExecutionRequest>,
     UnboundedReceiver<ExecutionResponse>,
     HashValue,
-    ExecutionPhase,
+    PipelinePhase<ExecutionRequest, ExecutionResponse>,
```

Comment:
we probably can use type alias later like type ExecutionPhase = PipelinePhase<ExecutionRequest, ExecutionResponse>